docs/src/components/home/Hero.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import ContentCopyRounded from '@material-ui/icons/ContentCopyRounded';
+import GradientText from 'docs/src/components/GradientText';
+
+const Hero = () => {
+  return (
+    <Box sx={{ overflow: 'hidden' }}>
+      <Container sx={{ minHeight: 500, height: 'calc(100vh - 120px)', maxHeight: 700 }}>
+        <Grid
+          container
+          alignItems="center"
+          wrap="nowrap"
+          sx={{ height: '100%', maxWidth: { xs: 500, md: 'initial' }, mx: 'auto' }}
+        >
+          <Grid item md={7} lg={6}>
+            <Box textAlign={{ xs: 'center', md: 'left' }} maxWidth={500}>
+              <Typography
+                variant="h1"
+                fontWeight={800}
+                fontSize="clamp(2.625rem, 1.2857rem + 3.5714vw, 4.5rem)"
+                lineHeight={1.11}
+                mb={2}
+              >
+                The <GradientText>ultimate</GradientText> solution for your UI.
+              </Typography>
+              <Typography variant="body1" color="text.secondary" mb={3}>
+                MUI provides robust, customizable, and accessible components, enabling you to build
```

Comment:
We might have a thickness issue with this font or contrast, I don't know, it doesn't feel comfortable when reading:

<img width="541" alt="Capture d’écran 2021-07-20 à 13 23 13" src="https://user-images.githubusercontent.com/3165635/126316083-0fc68fb9-1320-4ba4-8ab1-4d5c4f2b3073.png">

On Figma, it's OK

<img width="541" alt="Capture d’écran 2021-07-20 à 13 25 33" src="https://user-images.githubusercontent.com/3165635/126316222-1abf1f59-5ce6-41f2-a6fc-9e14a00b2dea.png">

Reply:
<img width="415" alt="Screen Shot 2021-07-20 at 11 48 02" src="https://user-images.githubusercontent.com/67129314/126345027-1935782e-64a8-47ca-86ba-08917962ca69.png">

On Webflow, the default value for the font-weight I've used is 500. I see that we're rendering as 400 and maybe that makes it a bit harder to read.

Reply:
For the width of the space character, is there anything we can do about it? Or do we even acknowledge that something is off?

Reply:
Well, I don't personally feel bothered by it. Maybe we could check with the rest of the team if this is something more alarming and then iterate on the letter-spacing, what do you think?

Reply:
More feedback sounds great. Maybe it's only me. The issue I get is that I not as straightforward as the GitHub UI's font for me to group characters into words.